pull_request_template.md

```diff
@@ -1,15 +1,16 @@
 #### Link to Jira: https://samdesk.atlassian.net/browse/DEV-*ISSUE_NUMBER*
 
-#### Change description
+#### Change description (can see Jira for full details, here we just need a jot note or two)
 - *todo*
 
 #### Linked PRs
-- *todo: Delete this section if there are no linked PRs*
+- *todo: Delete this section if there are no linked PRs. 
```

Comment:
Idle thought, we could `<!-- comment out -->` sections if we think they will commonly not be used? Then they wouldn't show up but can easily be put back in. I'm not sure how the backend side of the world works though so not sure if it is more common to have linked PRs or not. :) 

Can also be handy for just instructions that are helpful when writing the PR but don't ned to show in the visible verstion.

Reply:
I like that idea, though I think this one is commonly used. At least I end up using it more often than not at the moment. If we continue the move to a mono-repo on the JS side it should get less common.